### PR TITLE
Update Grant-SPOHubSiteRights.md

### PR DESCRIPTION
**Pull request by PowershellScripts**

### CODE REVIEW

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@get-itips @dariomws @tseward

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@dariomws @tseward

**Review by tseward — APPROVED** [review]

File: sharepoint/sharepoint-ps/sharepoint-online/Grant-SPOHubSiteRights.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ Accept wildcard characters: False
 
 ### -Principals
 
-One or more principles to add permissions for.
+One or more principals to add permissions for.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Grant-SPOHubSiteRights.md:65]:
```suggestion
One or more principals to add permissions for.
```

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**